.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: haskell
 
 ghc:
+ - 7.10
```

Comment:
Are you sure this works? I don't know that Travis has 7.10 support yet. Further, I'm pretty sure Travis parses the above the same as 7.1, so the version numbers should really be quoted.